src/Filter.js

```diff
@@ -0,0 +1,52 @@
+
+
+
+const Filter = (props) => {
+
+
+  return <div className={"container backdrop"}>
+    <div className={"taskModal"}>
+      <div className="justify-content-center">
+        <h3>Sort Tasks By</h3>
+      </div>
+      <div className={"row align-items-center"}>
+
+        <button className={"col-3 priority-button"} type={"button"}
```

Comment:
Lots of duplicated code (14-21 very similar to 23-30). Could remove duplicated code with a function.